### DISCUSSION TITLE
localhost域名支持

### DISCUSSION THREAD

**Discussion by realzolo** [1Panel-dev/1Panel#4881] [Q&A]
### 1Panel 版本

v1.10.6-lts

### 请描述您的需求或者改进建议

希望支持输入localhost域名
![image](https://github.com/1Panel-dev/1Panel/assets/63219216/49e76c33-1f7c-4b58-af01-3d262fe4cbd9)

**Comment by mobeicanyue** [comment]
为什么要用局域网才能使用的域名，1panel不是管理服务器的吗

**Comment by realzolo** [comment]
> 为什么要用局域网才能使用的域名，1panel不是管理服务器的吗

有没有可能是内网服务器呢？

**Comment by mobeicanyue** [comment]
内网可以自签证书和搞dns吧，你localhost不是只能本机访问

**Comment by realzolo** [comment]
> 内网可以自签证书和搞dns吧，你localhost不是只能本机访问

内网服务器 + FRP + 反代

**Comment by mobeicanyue** [comment]
反代直接 127.0.0.1:port 啊

**Comment by realzolo** [comment]
> 反代直接 127.0.0.1:port 啊

我当然知道直接 127.0.0.1:port 可以。我是希望localhost:port也支持

**Comment by mobeicanyue** [comment]
而且反向代理设置也不是在这里吧…不知道你想表达什么

**Comment by realzolo** [comment]
> 而且反向代理设置也不是在这里吧…不知道你想表达什么

...我这个截图只是说明这个域名输入框不能输入localhost这个域名，我又没说反代配置在这里。我不知道你是怎么理解的

**Comment by mobeicanyue** [comment]
你的意思是，外部访问 localhost 映射到你的服务器？

**Comment by realzolo** [comment]
> 你的意思是，外部访问 localhost 映射到你的服务器？

当然不是啊，不理解算了。。。

**Comment by realzolo** [comment]
> 你的意思是，外部访问 localhost 映射到你的服务器？

就比如这个demo.onezol.com这个站点，这个站点就是我部署在内网服务器上通过frp暴露在公网的。
<img width="1920" alt="1714924467260" src="https://github.com/1Panel-dev/1Panel/assets/63219216/ab9da9c2-9aed-4ac2-8fe3-cbd26680fa57">
为了在局域网和公网下都能访问到这个站点，这里必须配置一个“127.0.0.1:port”的域名(只配置demo.onezol.com是没什么用的)，不加这个局域网是无法访问的。
我想说的是，这里既然字面意义上说的是“域名”，我更希望配置的是“localhost:port”，而不是“ip+port”。

**Comment by zhengkunwang223** [comment]
这种建议还是自己修改配置文件
